config/schedule.rb

```diff
@@ -19,7 +19,7 @@
 
 # Learn more: http://github.com/javan/whenever
 
-every 6.hours do
+every 1.day, at: '2:00 am' do
```

Comment:
There's a split second where the faculty slides are repopulating themselves as the task completes. Even though it's very quick and the chances of it being seen are slim, it's best to just make it so that the job will never be seen.